test/powershell/Language/Scripting/NativeExecution/NativeStreams.Tests.ps1

```diff
@@ -1,70 +1,70 @@
-Describe "Native streams behavior with PowerShell" -Tags 'CI' {
```

Comment:
Not sure why the diff is showing up this way.  I made edits directly replacing EchoArgs in this case.